### PR TITLE
Clean out OpenMP remnants

### PR DESCRIPTION
**Pull request by mitchute**
Removes remnants of OpenMP work that was removed circa V8.0.

### CODE REVIEW

**Comment by mjwitte** [comment]
@mitchute Is a transition rule on your do-list for this?  Will need to remove any lingering ProgramControl objects.
Also, EP-Launch has an option for "Disable Multi-Threading" and some related lines in [Epl-run.bat](https://github.com/NREL/EnergyPlus/blob/79c5313d90a1d2e256d21793c9be7f33bf3b59b7/scripts/Epl-run.bat#L324-L328).  I suppose those should get a followup issue posted.

**Comment by mitchute** [comment]
@mjwitte I just added the transition rule. I will go ahead and post a followup issue for EP-Launch.

**Comment by Myoldmopar** [comment]
Thanks @mitchute for resolving the conflict.  I pulled develop **again** since it moved in the moments since you did your pull.

**Comment by Myoldmopar** [comment]
I will wait on CI to start on the current develop branch then we're good to go.

**Comment by Myoldmopar** [comment]
Well, frack.  Any idea what went wrong there @mitchute?  I was going to freeze the IDD right after merging this in.

File: idd/Energy+.idd.in [hunk 1]
```diff
@@ -20283,7 +20274,7 @@ SteamEquipment,
        \ip-units Btu/h-ft2
   N3 , \field Power per Person
        \type real
-       \minimum 0
+       \minimum
```
Comment (Myoldmopar) [inline idd/Energy+.idd.in:20277]: ?

Comment (mitchute) [inline idd/Energy+.idd.in:20277]: Blah. Something happened to my Notepad++ install, so I reinstalled a few days back. When I was fixing diffs in this I realized that my whitespace elimination macro wasn't working. This must have gotten deleted while I was testing that.

I'll put it back.

**Comment by Myoldmopar** [comment]
And there it is; looking good again.  Merging.

**Pull request merged**